apps/mull-api/src/schema.gql

```diff
@@ -32,6 +33,11 @@ input UpdateUserInput {
   name: String
 }
 
+"""
+The `Upload` scalar type represents a file upload.
+"""
```

Comment:
Add the following to the `.prettierignore` file:
```
schema.gql
```
And then run the backend to revert the fix prettier fix.